backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/coverage/CoverageRegionCollector.kt

```diff
@@ -38,13 +38,13 @@ internal class CoverageRegionCollector(private val fileFilter: (IrFile) -> Boole
         }
 
         override fun visitFunction(declaration: IrFunction) {
-            if (!declaration.isInline && !declaration.isExternal && !declaration.isGeneratedByCompiler) {
-                declaration.body?.let {
-                    val regionsCollector = IrFunctionRegionsCollector(fileFilter, file)
-                    regionsCollector.visitBody(it)
-                    if (regionsCollector.regions.isNotEmpty()) {
-                        functionRegions += FunctionRegions(declaration, regionsCollector.regions)
-                    }
+            if (!declaration.isInline && !declaration.isExternal && !declaration.isGeneratedByCompiler
+                    // TODO: Find better way to filter out `Konan_start`
+                    && declaration.name.asString() != "Konan_start") {
```

Comment:
Better to move this check under `isGeneratedByCompiler` property.

Reply:
Done.